Core/CoreTests/Database/Models/QuizTests.swift

```diff
@@ -40,4 +40,35 @@ class QuizTests: CoreTestCase {
         XCTAssertNil(quiz.viewableGrade)
         XCTAssertNil(quiz.viewableScore)
     }
+
+    func testAllowedAttemptsText() {
+        XCTAssertEqual(Quiz.make([ "allowedAttempts": -1 ]).allowedAttemptsText, "Unlimited")
+        XCTAssertEqual(Quiz.make([ "allowedAttempts": 1 ]).allowedAttemptsText, "1")
+        XCTAssertEqual(Quiz.make([ "allowedAttempts": 10 ]).allowedAttemptsText, "10")
+    }
+
+    func testQuestionCountText() {
+        XCTAssertEqual(Quiz.make([ "questionCount": 1 ]).questionCountText, "1")
+        XCTAssertEqual(Quiz.make([ "questionCount": 10 ]).questionCountText, "10")
+    }
+
+    func testNQuestionsText() {
+        XCTAssertEqual(Quiz.make([ "questionCount": 1 ]).nQuestionsText, "1 Question")
+        XCTAssertEqual(Quiz.make([ "questionCount": 10 ]).nQuestionsText, "10 Questions")
+    }
+
+    func testTimeLimitText() {
+        XCTAssertEqual(Quiz.make([ "timeLimitRaw": nil ]).timeLimitText, "None")
+        XCTAssertEqual(Quiz.make([ "timeLimitRaw": 10 ]).timeLimitText, "10min")
```

Comment:
This doesn't need a space?

Reply:
Dave would have preferred a space, but it comes with a comma when there are more parts: "1 hr, 30 min" is `.short` and `1hr 30min` is `.brief`. He preferred no comma.